bundle/manifests/storageosoperator.clusterserviceversion.yaml

```diff
@@ -49,190 +49,8 @@ spec:
         name: nfsservers.storageos.com
         version: v1
       - description: StorageOSCluster is the Schema for the storageosclusters API
-        displayName: StorageOS Cluster
         kind: StorageOSCluster
         name: storageosclusters.storageos.com
-        resources:
-        - kind: Namespace
```

Comment:
@darkowlzz I'm not sure why `make bundle` removed this - shall I revert?

Reply:
Yes, this is a known problem in the existing bundle generation, which is based on some old tools. For now, this block has to be added back. The new tooling has much better way to generate and populate this using kustomize and some code markers.

Reply:
thanks - done